lib/improver/spotdata/neighbour_finding.py

```diff
@@ -192,6 +192,10 @@ def fast_nearest_neighbour(cube, sites, orography=None):
             dz_site_grid = 0.
             if orography is not None:
                 dz_site_grid = altitude - orography[i_latitude, j_longitude]
+            # If site altitude set with fill value, assume it is on surface of
+            # of model orography at equivalent altitude to nearest neighbour.
```

Comment:
Okay, I created a shared_constants class in common_functions and grab it from there in both site_data and neighbour_finding.